clients/src/main/java/org/apache/kafka/common/security/oauthbearer/internals/secured/ConfigurationUtils.java

```diff
@@ -228,4 +233,16 @@ public <T> T get(String name) {
         return (T) configs.get(name);
     }
 
+    // make sure the url is in the "org.apache.kafka.sasl.oauthbearer.allowed.url" system property
+    public void throwIfURLIsNotAllowed(String urlConfig) {
+        Set<String> allowedLoginModuleList = Arrays.stream(
+                        System.getProperty(ALLOWED_SASL_OAUTHBEARER_URL_CONFIG, "").split(","))
```

Comment:
If we default to empty list, is this going to break all users and force them to set this new system property? If so we should probably default to allow all paths, no?

Reply:
As discussed offline, we'll default to empty list in 4.0.